js/templates/userPage/home.html

```diff
@@ -56,8 +56,10 @@
         <% } %>
         <% ob.contactInfo.social.forEach(function(account) { %>
           <div class="listItem">
-            <div><%= ob.polyT('properNames[account.type]') %></div>
-            <div class="clrT2"><%= account.username %></div>
+            <div><%= account.type %></div>
+            <div class="clrT2">
+              <% print(ob.is.url(account.username) ? `<a href="${account.username}">${account.username}</a>` : account.username) %>
```

Comment:
Could we add a `data-open-external` to that anchor? If they include a non-ob protocol (e.g. http://), then it could detect external, but if they just put `www.espn.com`, it thinks that's an internal link.